src/index.html

```diff
@@ -1,14 +1,19 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Colored blocks</title>
   <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-  <h1>Colored blocks</h1>
+    <div class="upBlock"></div>
```

Comment:
Follow the indentation rules. Add an additional space of 2 spaces for nested tags.
https://mate-academy.github.io/style-guides/htmlcss.html#indentation